pkg/syncer/storagepool/storageclasswatch.go

```diff
@@ -328,7 +332,7 @@ func (w *StorageClassWatch) fetchPolicies(ctx context.Context) ([]cnsvsphere.Spb
 	log := logger.GetLogger(ctx)
 	var profiles []cnsvsphere.SpbmPolicyContent
 	var err error
-	policyIds := w.policyIds
+	policyIds := w.policyIDs
 	for i := 0; i < 2; i++ {
```

Comment:
why do we have for loop here why we have this check ` i < 2`?

Reply:
Looking at the function description, if the first `PbmRetrieveContent` call fails, it returns am empty response. Then, we identify which profiles are invalid, remove them and retry the operation.